_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "API versions conversion"
```

Comment:
Maybe we can also come up with some better title? _Path to CRD v1_? _Changes to Strimzi API versions_?

Reply:
`Path to CRD v1` sounds good to me :)